VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.spec.ts

```diff
@@ -589,18 +590,26 @@ describe('PrivateConsultationParticipantsComponent', () => {
             expect(result).toEqual([]);
         });
 
-        it('should return nothing if is not joh consultation', () => {
+        it('should return list in correct order for joh consultation', () => {
```

Comment:
Need to confirm the order. Observers and quick link observers are grouped together and ordered by display name currently. Need to double check if they should be separated or not.